stellar/deploy-contract.js

```diff
@@ -145,11 +150,8 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
             const itsHubAddress = nativeToScVal(config.axelar?.contracts?.InterchainTokenService?.address, { type: 'string' });
             const chainName = nativeToScVal(chain.axelarId, { type: 'string' });
             const nativeTokenAddress = nativeToScVal(Address.fromString(chain?.tokenAddress), { type: 'address' });
-
-            const interchainTokenWasmFile = await getWasmFile(options, 'interchain_token');
-            const interchainTokenWasmUploaded = await uploadWasm(interchainTokenWasmFile, wallet, chain);
-            const tokenManagerWasmFile = await getWasmFile(options, 'token_manager');
-            const tokenManagerWasmUploaded = await uploadWasm(tokenManagerWasmFile, wallet, chain);
+            const interchainTokenWasmUploaded = await uploadContract('interchain_token', options, wallet, chain);
```

Comment:
```suggestion
            const interchainTokenWasmHash = bytesToScVal(await uploadContract('interchain_token', options, wallet, chain));
```
create a conversion helper and avoid the need for these intermediate variables

Reply:
make sense, created `wasmHashBytesToScVal`